.github/workflows/test.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         env:
           OCP_SERVER_URL: ${{ secrets.OCP_SERVER_URL }}
           OCP_TOKEN: ${{ secrets.OCP_TOKEN }}
-          OCP_NAMESPACE: vscode-${{ github.ref_name }}-${{ runner.os }}
+          OCP_NAMESPACE: vscode-${{ github.head_ref }}-${{ runner.os }}
```

Comment:
Sorry, looks like we missed a line. Please add this change here also https://github.com/IBM/operator-collection-sdk-vscode-extension/blob/28a222a15a6598f3ae29b4d23edf68d5517dd2ed/.github/workflows/test.yml#L80